openedx/features/badging/admin.py

```diff
@@ -1,7 +1,10 @@
 from django.contrib import admin
 from .models import Badge, UserBadge
 
-admin.site.register(Badge)
+
+@admin.register(Badge)
+class BadgeAdmin(admin.ModelAdmin):
+    list_display = ('name', 'type', 'threshold')
```

Comment:
Did we have a discussion with client on what fields to display on admin side?

Reply:
It will make user experience better, So it doesn't require client discussion.